src/redux/Actions.js

```diff
@@ -0,0 +1,23 @@
+//import Reducer from './Reducer'; 
+import { getWeather } from './ActionTypes';
+import axios from 'axios';
+
+export const fetchWeather = (city) => {
+  return (dispatch) => {
+      dispatch({ type: getWeather.PENDING });
+       axios.get(`https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=metric&appid=d804734ee459510001eab20076e914f2`)
```

Comment:
You should have the appId in an environment variable. If the curriculum hasn't gone over it yet, you can lookup some vids on how to do it. API keys should be kept secret, and they help you do that.